tests/test_fixture_use.py

```diff
@@ -0,0 +1,27 @@
+def test_basic_use_fixtures(suite, suite_test):
+    fixtures_names = []
+    events_added = []
+    for _ in range(3):
+        fixture = suite_test.file.add_fixture()
+        events_added.append(fixture.add_event())
+        fixtures_names.append(fixture.name)
+    suite_test.add_decorator('slash.use_fixtures({})'.format(fixtures_names))
+    events_happened = suite.run().events
+    for event in events_added:
+        assert event in events_happened
+
+def test_nested_fixtures(suite, suite_test):
+    nested_fixture = suite_test.file.add_fixture()
+    fixture = suite_test.file.add_fixture()
+    fixture.depend_on_fixture(nested_fixture)
+    suite_test.add_decorator('slash.use_fixtures({})'.format([fixture.name]))
+    suite.run()
+
+def test_use_fixture_and_depend_on_it_not_allowd(suite, suite_test):
+    fixture = suite_test.file.add_fixture()
+    suite_test.depend_on_fixture(fixture)
+    suite_test.add_decorator('slash.use_fixtures({})'.format([fixture.name]))
+    suite_test.expect_failure()
+    summary = suite.run()
+    [result] = summary.get_all_results_for_test(suite_test)
+    assert "using twice fixtures" in result.get_failures()[0].message
```

Comment:
Need to add a test making sure the cleanup of the extra fixtures gets called... this is a major concern in such a feature